jupyter_config.json

```diff
@@ -1,4 +0,0 @@
-{
-  "LabApp": { "collaborative": true, "expose_app_in_browser": true },
-  "JupyterNotebookApp": { "collaborative": true, "expose_app_in_browser": true }
```

Comment:
Probably we want to keep this file, so RTC can be tested on Binder?

Reply:
Restored